### PR TITLE
Added luna Lua scripting library

### PR DESCRIPTION
**Pull request by smallfx**

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -4889,5 +4889,17 @@
     "web": "https://github.com/miere43/nim-registry",
     "license": "MIT",
     "method": "git"
+  },
+  {
+	"name": "luna.nim",
```
Comment (dom96) [inline packages.json]: Spaces instead of tabs please :)

Comment (smallfx) [inline packages.json]: @dom96 my bad! Fixed now.

File: packages.json [hunk 2]
```diff
@@ -4889,5 +4889,17 @@
     "web": "https://github.com/miere43/nim-registry",
     "license": "MIT",
     "method": "git"
+  },
+  {
+    "name": "luna.nim",
```
Comment (dom96) [inline packages.json]: Sorry, one more thing. You should just call your package ``luna``. Rename your ``.nimble`` file to ``luna.nimble`` as well and make sure the ``name`` field inside is set to ``luna``.

Comment (dom96) [inline packages.json]: The name of your github repo doesn't need to be renamed.

Comment (smallfx) [inline packages.json]: @dom96 changed it :)

**Comment by dom96** [comment]
Thanks!

**Pull request merged**